modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -128,8 +129,10 @@ export class BitGoAPI implements BitGoBase {
   protected readonly _clientId?: string;
   protected readonly _clientSecret?: string;
   protected _validate: boolean;
+  public readonly allowSendingCookies: boolean;
```

Comment:
```suggestion
  public readonly enableCookies: boolean;
```
More clear in intent

Reply:
Yeah, I also feel like allowSendingCookies is both long and vagueish so I am not satisfied with the name.
On the other hand `enableCookies` may sound like the SDK uses cookies to implement some internal logic, which is not the case. This change is to just allow the SDK user to send cookies with the request if they want this. Should it be `enableCookiesPropagation`?